### DISCUSSION TITLE
Convert Encoding from Windows 1250 to UTF8

### DISCUSSION THREAD

**Discussion by OSch24** [microsoft/azuredatastudio#17315] [Q&A]
Hello, I am looking for a way to convert an existing SQL script to UTF 8. 

I often have old scripts still in the format "Windows 1250 / 52" etc. and would like to convert them to UTF8.

So far I have not been able to find an addon. Is there this possibility or would this be something for a function request?

thanks. Oliver

**Comment by Charles-Gagnon** [comment]
At the bottom of the window you should see an item with the encoding : e.g. 

![image](https://user-images.githubusercontent.com/28519865/136594120-f6298345-da28-4c12-a2cd-275fb7295301.png)

If you click that then you should see an option to save it with a different encoding.

**Reply by OSch24** [reply]
I am aware of that.
But no conversion takes place. 
The file is then "only" opened again with this coding.

**Reply by Charles-Gagnon** [reply]
What kind of conversion are you expecting? This isn't really my area of expertise - but it looks like detecting whether a file is UTF-8 or one of the 1250/52 isn't straightforward - see https://github.com/microsoft/vscode/issues/133075 for some discussion around the issue.

How are you checking whether it was saved as you expected?

**Reply by OSch24** [reply]
Thank you for your help.

The problem we have is that some scripts are in Windows 1250 code. Since there are always problems with umlauts (ä, ü, ö), I would like to "convert" these scripts to UTF-8.

I am looking for a way to do this in ADS.
